src/Microsoft.Sbom.Api/Executors/ILicenseInformationFetcher.cs

```diff
@@ -17,12 +17,16 @@ public interface ILicenseInformationFetcher
     /// <returns></returns>
     List<string> ConvertComponentsToListForApi(IEnumerable<ScannedComponent> scannedComponents);
 
+    /// <inheritdoc cref="FetchLicenseInformationAsync(List{string}, int)"/>>
+    Task<List<string>> FetchLicenseInformationAsync(List<string> listOfComponentsForApi);
```

Comment:
Good choice keeping the old method intact--is there a way to specify that we're using a default timeout here?

Reply:
Thanks. I updated the method docs to say that a default timeout is used